### PR TITLE
Election administration and frontend 

### LINKED ISSUE #5: Create a new election
Ref.: VOC-ADM-01

As an administrator I want to be able to create an Election in Decidim for voting with [Vocdoni](https://vocdoni.io).
 
Note that Decidim already have a module and component called Election, so we'll need to take this into account (ie using the `Decidim::Vocdoni` namespace, the module name will be `decidim-module-vocdoni`, the component name will be "`Vocdoni`" and similar approaches).

For the metadata we'll need the same attributes supported by Vocdoni API, such as: 

1. Title: text with i18n
2. Description: textearea with i18n
3. Image header: image
4. Stream URI: URL
5. Start Date: datetime
6. End Date: datetime

- [ ] As an administrator I can create a new component with this metadata
- [ ] As an administrator I can update this component
- [ ] As an administrator I can publish and unpublish this component
- [ ] As an administrator I can delete this component

### PR DESCRIPTION
**Pull request by andreslucena**
This PR add the initial CRUD for Elections. It was extracted and adapted from `decidim-elections`. 

By adapted I mean mostly: 

1. Removing unnecessary features (like Trustees) 
2. Changing the namespace from Elections to Vocdoni
3. Changing a bit the data model to adapt it to our needs

In the meantime I found a couple of bugs that I'd need to check out if I can reproduce them with other module, as they seem like they were bugs on the generator (like 2dcdc9580f300300f31c67416439244b40911e25 ee73af5cffab3fb2e661b1ce02b0f397672f4808 1fbde177c6086f29a0dbc70185963b898c311d96 5393b792031209d2387f63323a55e6c1c4ca7489)

Finally as this is also adding lots of specs, I've added the GH Actions CI configuration. 

Fixes #5 

## Testing

- [ ] As an administrator I can create a new component with this metadata
- [ ] As an administrator I can update this component
- [ ] As an administrator I can publish and unpublish this component
- [ ] As an administrator I can delete this component

### CODE REVIEW

**Review by microstudi — APPROVED** [review]
Everything good @andreslucena , nice job!

A part from a little thing in the readme, do you thing we should rename the component to something like "Vocdoni Elections"? I think that seeing only "Vocdoni" might be confusing for admins.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
 
 ## Installation
 
+This module is only compatible with Decidim v0.27.
+
 Add this line to your application's Gemfile:
 
 ```ruby
-gem "decidim-vocdoni"
+gem "decidim-vocdoni", github: "andreslucena/decidim-module-vocdoni"
```
[the remark this answers is not in the document]
Comment (andreslucena) [reply README.md:26]: Nice catch, this is from before I created this organization

**Comment by andreslucena** [comment]
> Everything good @andreslucena , nice job!
> 
> A part from a little thing in the readme, do you thing we should rename the component to something like "Vocdoni Elections"? I think that seeing only "Vocdoni" might be confusing for admins.

I agree. Its done in e1f72c6572bec8a96622ab2d2872177c087c283c

I think in my first version I had this name but I didn't like that most of the other modules have only one word and this would break a bit the UI. Seeing it with a fresh pair of eyes, I see that actually "Elections (Vocdoni)" isn't that far from "Accountability" (20 vs 15 characters). This is how it looks like: 

Selection_490

**Review by microstudi — APPROVED** [review]
great, lets go!

**Pull request merged**